patches/tippy.js@6.3.7.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/dist/tippy.esm.js b/dist/tippy.esm.js
+index 55346f57874b0af79fec3b6f9f0253f9d55bfee2..c7a4e1549dcc3e1815c908825d914d44467e60f3 100644
+--- a/dist/tippy.esm.js
++++ b/dist/tippy.esm.js
+@@ -1907,7 +1907,7 @@
+   var normalizedReturnValue = normalizeToArray(returnValue);
+ 
+   function onTrigger(event) {
+-    if (!event.target || disabled) {
++    if (!isElement(event.target) || disabled) {
+       return;
+     }
```

Comment:
Generated this diff by using:
`diff -u node_modules/tippy.js/dist/tippy.esm.js node_modules/tippy.js/dist/tippy.esm.modified.js > patches/tippy.js@6.3.7.patch`

Where I added the changes I wanted in `tippy.esm.modified.js` and `tippy.esm.js` is the original file.

Reply:
You aren’t expected to do any of this manually. See [`pnpm patch`](https://pnpm.io/cli/patch).